### PR TITLE
Add ternary autofix eslint rule and unused vars

### PR DESCRIPTION
**Pull request by Tbaut**
Subsequent to https://github.com/ChainSafe/files-ui/pull/1074#pullrequestreview-689246755

### CODE REVIEW

File: packages/common-components/src/TextInput/FormikTextInput.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const FormikTextInput = React.forwardRef(
         name={field.name}
         value={field.value}
         placeholder={placeholder}
-        captionMessage={meta.error ? `${meta.error}` : captionMessage ? captionMessage : null}
+        captionMessage={meta.error ? `${meta.error}` : captionMessage || null}
```
Comment (tanmoyAtb) [inline packages/common-components/src/TextInput/FormikTextInput.tsx]:
```suggestion
        captionMessage={meta.error || captionMessage || null}
```

File: packages/common-components/src/NumberInput/FormikNumberInput.tsx [hunk 2]
```diff
@@ -61,7 +61,7 @@ const FormikNumberInput: React.FC<FormikNumberInputProps> = ({
       value={field.value}
       placeholder={placeholder}
       captionMessage={
-        meta.error ? `${meta.error}` : captionMessage && captionMessage
+        meta.error ? meta.error : captionMessage && captionMessage
```
Comment (tanmoyAtb) [inline packages/common-components/src/NumberInput/FormikNumberInput.tsx]:
```suggestion
        meta.error || captionMessage || null
```

**Review by tanmoyAtb — APPROVED** [review]
Thanks, this was much needed!

File: packages/common-components/src/TextInput/FormikTextInput.tsx (hunk 1 above)
Comment (Tbaut) [reply packages/common-components/src/TextInput/FormikTextInput.tsx]: Thanks! Pushed a fix, the null is actually not even needed.

**Review by RyRy79261 — APPROVED** [review]

**Pull request merged**